modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -544,16 +548,60 @@ private CompletableFuture<?> onTableCreate(ConfigurationNotificationEvent<TableV
         }
 
         try {
-            return createTableLocally(
+            DistributionZoneView zone =
+                    getZoneById(distributionZonesConfiguration, (ctx.newValue()).zoneId()).value();
+
+            List<Set<Assignment>> assignments = AffinityUtils.calculateAssignments(
+                    baselineMgr.nodes().stream().map(ClusterNode::name).collect(toList()),
```

Comment:
I think that we shouldn't use here baseline manger, what do you think about `DistributionZoneManager#topologyVersionedDataNodes` while proper solution with causality token is not implemented? 


In any cases, we need to add todo here to use proper method

Reply:
I see that you've removed usage of `DistributionZoneManager#topologyVersionedDataNodes` in a further commit, so in general I'm ok with using `baselineMgr.nodes()` and adding to do